### PR TITLE
ENT-9757: Main navigation width adjustment

### PR DESCRIPTION
**Pull request by mineralsfree**
Ticket: ENT-9757
Changelog: None

### CODE REVIEW

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/426/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/426/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-426/output/_site/

**Comment by mineralsfree** [comment]
@cf-bottom jenkins, 🥺

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/427/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/427/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-427/output/_site/

**Comment by mineralsfree** [comment]
@cf-bottom jenkins, 🙏 🥺

**Comment by mineralsfree** [comment]
@cf-bottom jenkins, build

**Comment by mineralsfree** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/432/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/432/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-432/output/_site/

**Comment by mineralsfree** [comment]
Got an approve from Chiacheng about the implementation

File: generator/_assets/styles/less/variables.less [hunk 1]
```diff
@@ -32,4 +32,4 @@
 @desktop-wide-down: ~'(max-width: 1439px)';
 @desktop-down: ~'(max-width: 1279px)';
 @tablet-down: ~'(max-width: 1023px)';
-@phone-down: ~'(max-width: 589px)';
+@phone-down: ~'(max-width: 589px)';
```
Comment (aleksandrychev) [inline generator/_assets/styles/less/variables.less:35]: please add new line to the end of the file

File: generator/_assets/styles/less/header.less [hunk 2]
```diff
@@ -41,7 +41,8 @@
       flex-direction: column;
       background: #FFFFFF;
       z-index: 10;
-      width: 24.3rem;
+      width: clamp(243px, 50%, 433px);
+      min-width: 24.3rem;
```
Comment (aleksandrychev) [inline generator/_assets/styles/less/header.less:45]: no need to use min-width as clam's first arg is responsible for it

File: generator/_assets/styles/less/header.less [hunk 3]
```diff
@@ -41,7 +41,8 @@
       flex-direction: column;
       background: #FFFFFF;
       z-index: 10;
-      width: 24.3rem;
+      width: clamp(243px, 50%, 433px);
```
Comment (aleksandrychev) [inline generator/_assets/styles/less/header.less:44]: please use rem units, in our case 1rem = 10px

File: generator/_assets/styles/less/variables.less (hunk 1 above)
Comment (mineralsfree) [reply generator/_assets/styles/less/variables.less]: fixed

File: generator/_assets/styles/less/header.less (hunk 3 above)
Comment (mineralsfree) [reply generator/_assets/styles/less/header.less]: fixed

**Review by aleksandrychev — APPROVED** [review]

**Pull request merged**